bin/kubeconfig.bash

```diff
@@ -85,7 +87,7 @@ case "${1}" in
         user_kubeconfig=${CK8S_CONFIG_PATH}/user/secret/kubeconfig.yaml
     ;;
     dev)
-        serviceAccount="${2}"
+        serviceAccount="${2?Missing argument service account}"
         if [ -z "${serviceAccount}" ]; then
           echo "Error: Service account name is needed" >&2
           usage
```

Comment:
What is the point of this? It won't ever trigger if I'm not mistaken?

Other than that, I tested the changes and it looks good.

Reply:
I seem to have overlooked the `if` there, because what I got was `bin/kubeconfig.bash: line 119: 2: unbound variable` because of `set -u`.